documentation/writing_yaml_plans.md

```diff
@@ -189,7 +189,9 @@ Script steps use these fields:
 -   `script`: The script to run
 -   `targets`: A target or list of targets to run the script on
 -   `arguments`: (Optional) An array of command-line arguments to pass to the
-    script
+    script. Mutually exclusive with `pwsh_params`.
```

Comment:
I'll let Heston rule on this, but I'm not sure everyone will know what 'mutually exclusive' means? "Can't be used with" might be friendlier.